pulsar-client/src/main/java/org/apache/pulsar/client/impl/TypedMessageBuilderImpl.java

```diff
@@ -20,6 +20,7 @@
 
```

Comment:
The changes of this file are not needed anymore

Reply:
Removed the change